app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+
+  def create
+    @post = Post.find(params[:post_id])
+    
+    @comment = @post.comments.create(comment_params)
```

Comment:
It saves by adding it to @post then @post is being updated, or at least that's what it looks like it is doing.